src/templates/manifest.ts

```diff
@@ -0,0 +1,28 @@
+import { EncodedExtension } from '@openshift/dynamic-plugin-sdk-webpack';
+import { ConsolePluginBuildMetadata } from '@openshift-console/dynamic-plugin-sdk-webpack/lib/build-types';
+
+import type { YAMLTemplate } from '@openshift-console/dynamic-plugin-sdk';
+
+const NetworkAttachmentDefinitionExtensionModel = {
+  version: 'v1',
+  group: 'k8s.cni.cncf.io',
+  kind: 'NetworkAttachmentDefinition',
+};
```

Comment:
Don't you get NetworkAttachmentDefinitionModelGroupVersionKind from kubevirt-api?

Reply:
Ts-node throws an error

```
import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

I should further investigate with Vojtech